lib/src/main/java/com/auth0/jwt/impl/JWTParser.java

```diff
@@ -47,19 +67,11 @@ static ObjectMapper getDefaultObjectMapper() {
         return mapper;
     }
 
-    @SuppressWarnings("WeakerAccess")
-    <T> T convertFromJSON(String json, Class<T> tClazz) throws JWTDecodeException {
-        if (json == null) {
-            throw exceptionForInvalidJson(null);
-        }
-        try {
-            return mapper.readValue(json, tClazz);
-        } catch (IOException e) {
-            throw exceptionForInvalidJson(json);
-        }
+    private static JWTDecodeException decodeException() {
```

Comment:
there should be no reason to make this 2 methods `static`. Does this have any impact on the performance?